toolchain/sem_ir/facet_type_info.cpp

```diff
@@ -94,7 +94,9 @@ static auto SubtractSorted(
 
 auto FacetTypeInfo::Combine(const FacetTypeInfo& lhs, const FacetTypeInfo& rhs)
     -> FacetTypeInfo {
-  FacetTypeInfo info = {.other_requirements = false};
+  FacetTypeInfo info = {.special_requirements_mask =
+                            lhs.special_requirements_mask.value |
+                            rhs.special_requirements_mask.value};
```

Comment:
Consider popping this out as a separate statement? Perhaps down where other_requirements was being accumulated below before this change?
```suggestion
  FacetTypeInfo info = {};
  info.special_requirements_mask =
      lhs.special_requirements_mask.value |
      rhs.special_requirements_mask.value;
```

Reply:
Done.